docs/release.md

```diff
@@ -38,40 +23,13 @@ It should look like:
     $ echo $VERSION
     ```
 
-2. Make a branch to release
+2. Add a git tag to the main HEAD, then push it.
 
     ```console
-    $ git neco dev "bump-$VERSION"
+    git checkout main
+    git tag -a -m "Release v$VERSION" "v$VERSION"
```

Comment:
It would be good to ensure the freshness of the local HEAD.

```suggestion
    git checkout main
    git pull
    git tag -a -m "Release v$VERSION" "v$VERSION"
```